application/app/Http/Controllers/Admin/ProductCategoryController.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use App\Models\ProductCategory;
+use App\Models\Product;
+use App\Http\Requests\ProductCategory\IndexRequest;
+use App\Http\Requests\ProductCategory\StoreRequest;
+use App\Http\Requests\ProductCategory\UpdateRequest;
+
+class ProductCategoryController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index(IndexRequest $request)
+    {
+        $productCategories = ProductCategory::nameSearch($request->input('name'))
+                                            ->sortOrder($request->input('sortType'), $request->input('sortOrder'))
+                                            ->searchPaginate($request->input('display'));
+
+        return view('admin.product_categories.index', ['productCategories' => $productCategories]);
```

Comment:
```suggestion
        return view('admin.product_categories.index', compact('productCategories'));
```
これでいい
compactが何してるかは自分で調べてください